src/rprblender/export/mesh.py

```diff
@@ -197,14 +200,14 @@ def init_from_shape_type(shape_type, size, size_y, segments):
         finally:
             bm.free()
 
+# Create a queue for batched material synchronization tasks
+material_sync_queue = queue.Queue()
```

Comment:
Lifetime of material_sync_queue is extended to lifetime of object itself.
Old object lifetime was only for method call. 

Is it correct behavior?